### ISSUE TITLE
Create full defenition of bro lottery

### ISSUE THREAD

**Issue by bro-n-bro-0** [bro-n-bro/Bro_n_Bro-Wednesday#1]
- subject
- workflow
- technical side
- conclusions

**Comment by bro-n-bro-99** [comment]
TL;DR part

Bro_n_Bro IBC lottery is a number of events aimed to distribute IBC tokens between delegators Bro_n_Bro Validator. IBC lottery is taking action on every Wednesday starting Feb 16th. Distribution planning is a systematic approach of 12 rounds divided into 4 tours (3 rounds per tour) with 10 winners per round. The winners of each round will be defined by random picking python script and will be blacklisted till the next tour, in this way giving other participants a chance to win. To raise your winning chances delegate your assets to Bro_n_Bro Validator from one Keplr account to different chains - the more you delegate, the higher chances to win! There is a list of rewards distribution for each round. No need to claim the reward, transactions will be proceeded automatically. No registration, KYC or any other actions required. The IBC lottery is created with a purpose of cosmos network growth.

**Comment by bro-n-bro-0** [comment]
To avoid attacks on lottery it's defined to limit participants from the bottom. Top 500 sounds great
